README.md

```diff
@@ -346,8 +346,24 @@ lookup (for physical measurements and biospecimen donation) as well as a
 Participant Work Queue. Any of the above parameters can be provided as a query parameter to do
 an exact match. Examples:
 
-    GET /ParticipantSummary?hpoId=PITT
-    GET /ParticipantSummary?hpoId=PITT&state=PIIState_MA
+    GET /ParticipantSummary?awardee=PITT
+    GET /ParticipantSummary?awardee=PITT&state=PIIState_MA
+
+    GET /ParticipantSummary?organization=PITT_FOO 
```

Comment:
We should probably use real organization and site values for this... you can pull them from:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/data/organizations.csv
https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/data/sites.csv